modules/vector-search/pages/fine-tune-vector-search.adoc

```diff
@@ -0,0 +1,132 @@
+= Fine-tuning Vector Searches
+:page-topic-type: concept
+:page-ui-name: {ui-name}
+:page-product-name: {product-name}
+:stem:
+:description: pass:q[Additional parameters can be added to a vector search REST call that will allow you to tune the search for recall and/or accuracy.]
+
+[abstract]
+{description}
+
+[#overview]
+== Overview
+
+Couchbase automatically dynamically tunes vector indexes to achieve a balance between recall (the quality of the search results),
+latency (search response time), and memory efficiency.
+This tuning occurs during indexing and querying, eliminating the need for users to adjust optimization parameters manually.
+
+Couchbase dynamically adjusts two critical vector parameters:
+
+`nlist` (or `Centroid` count)::
+The number of clusters used for indexing.
+The centroids are used to quickly find the surrounding closest matches in the index. Increasing the number of centroids will increase accuracy but will decrease the speed of the search.
++
+The `nlist` is determined dynamically based on the size of the dataset (number of vectors) in a partition:
++
+[%header, cols="3*a"]
+|===
+| Number of vectors in partition (`nvec`)
+| `nlist` calculation
+| Notes
+
+| stem:["nvec " ge " 200,000"]
+| stem:[4 xx sqrt("nvec")]
```

Comment:
Can you build a preview site, please? I want to see how this renders in the table.